lua/telescope/pickers.lua

```diff
@@ -1210,6 +1293,11 @@ function Picker:_do_selection(prompt)
   end
 end
 
+--- Wrapper function for `Picker:new` that incorporates user provided `opts`
+--- with the telescope `defaults`
+---@param opts table
+---@param defaults table
+---@return any
```

Comment:
There are technically class annotation that you can do and then you can say `---@return Picker` and we can document fields for that class and so much more but i think that works to :laughing: